<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>200</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Сменить источник сырья</string>
     </property>
     <addaction name="action_3"/>
     <addaction name="action_4"/>
    </widget>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Базы данных</string>
     </property>
     <addaction name="actCreateDatabase"/>
     <addaction name="actChangeDatabase"/>
     <addaction name="actSyncDatabase"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="menu_3"/>
    <addaction name="actOptions"/>
    <addaction name="actConsole"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Отгрузки</string>
    </property>
    <addaction name="actNewOtgruska"/>
    <addaction name="actChangeOtgruzka"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>ДС</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <action name="actExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Давальческое сырье</string>
   </property>
  </action>
  <action name="action_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Собственное сырье</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Архивировать базу данных...</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Восстановить базу данных...</string>
   </property>
  </action>
  <action name="actOptions">
   <property name="text">
    <string>Настройки...</string>
   </property>
  </action>
  <action name="actConsole">
   <property name="text">
    <string>Консоль логов</string>
   </property>
  </action>
  <action name="actCreateDatabase">
   <property name="text">
    <string>Создать новую...</string>
   </property>
  </action>
  <action name="actChangeDatabase">
   <property name="text">
    <string>Сменить...</string>
   </property>
  </action>
  <action name="actSyncDatabase">
   <property name="text">
    <string>Синхронизировать с системой</string>
   </property>
  </action>
  <action name="actNewOtgruska">
   <property name="text">
    <string>Новая отгрузка</string>
   </property>
  </action>
  <action name="actChangeOtgruzka">
   <property name="text">
    <string>Выбрать отгрузку</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
